### PR TITLE
Post split cleanup

### PR DESCRIPTION
**Pull request by blairconrad**
Contributes to FakeItEasy/FakeItEasy#1735.

### CODE REVIEW

File: FakeItEasy.build [hunk 1]
```diff
@@ -1,240 +0,0 @@
...
-		<MakeDir Directories="$(ArchiveDirectory)" />
-
-		<!-- Archive-->
-		<CreateItem Include="$(BinariesPath)\Merged\*.*;$(BinariesPath)\Merged-SL\*.*;;$(LicensePath)\*.*">
-			<Output ItemName="archiveFiles" TaskParameter="Include" />
-		</CreateItem>
-
-		<Zip Files="@(archiveFiles)" ZipFileName="$(ArchiveDirectory)\$(ProjectName)-$(VersionNumber).zip" Flatten="true" WorkingDirectory="$(BuildPath)" />
-	</Target>
-</Project>
```
Comment (thomaslevesque) [inline FakeItEasy.build]: Wow. That monstrosity must be pretty old, I've never seen it before.

File: build.sh [hunk 2]
```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 set -euo pipefail
-dotnet run --project "./tools/FakeItEasy.Build/FakeItEasy.Build.csproj" -- $@
+dotnet run --project "./tools/FakeItEasy.Analyzers.Build/FakeItEasy.Analyzers/Build.csproj" -- $@
```
Comment (thomaslevesque) [inline build.sh]: FakeItEasy.Analyzers/Build => FakeItEasy.Analyzers.Build

Comment (blairconrad) [reply build.sh]: *gasp*

File: FakeItEasy.build (hunk 1 above)
Comment (blairconrad) [reply FakeItEasy.build]: I'm not saying I've never seen it, but I don't remember it!

**Comment by blairconrad** [comment]
I amended, like a savage. Running out now. Hope it works! Sorry about the `/`. :blush:.

**Comment by thomaslevesque** [comment]
Thanks @blairconrad!

**Comment by blairconrad** [comment]
Thank you!

**Pull request merged**